### ISSUE TITLE
ImportError: No module named models

### ISSUE THREAD

**Issue by AlessandroFazenda** [olipo186/Git-Auto-Deploy#165]
Info about my enviroment:
System: CentOS 6.8 (out-of-the-box minimal installation)
GIT: 2.11.0 (from IUS repo)
Python: 2.7.13 (from IUS repo)

After cloning the project from this git using the master revision (commit 465cb92) and trying to run, it throws the exception "ImportError: No module named models"
`[root@localhost Git-Auto-Deploy]# python2.7 -m gitautodeploy --config config.json
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 163, in _run_module_as_main
    mod_name, _Error)
  File "/usr/lib64/python2.7/runpy.py", line 111, in _get_module_details
    __import__(mod_name)  # Do not catch exceptions initializing package
  File "gitautodeploy/__init__.py", line 4, in <module>
    from .cli import *
  File "gitautodeploy/cli/__init__.py", line 2, in <module>
    from .models import *
ImportError: No module named models`

But the folder `models` is located under `gitautodeploy/` not under `gitautodeploy/cli/`. After edit the file `gitautodeploy/cli/__init__.py` and changed the line `from .models import *` to `from ..models import *` it throws a new exception "ImportError: No module named models"
`[root@localhost Git-Auto-Deploy]# python2.7 -m gitautodeploy --config config.json
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/Git-Auto-Deploy/gitautodeploy/__main__.py", line 7, in <module>
    main()
  File "gitautodeploy/gitautodeploy.py", line 639, in main
    init_config(config)
  File "gitautodeploy/cli/config.py", line 345, in init_config
    from .models import Project
ImportError: No module named models`

Edited file `gitautodeploy/cli/config.py` and changed the line `from .models import Project` to `from ..models import Project` and now the project runs fine.
`[root@localhost Git-Auto-Deploy]# python2.7 -m gitautodeploy --config config.json --allow-root-user
2017-01-13 13:10:02,189 [INFO ]  Cloning repository /root/repositories/Git-Auto-Deploy
2017-01-13 13:10:10,038 [ERROR]  Cloning into '/root/repositories/Git-Auto-Deploy'...
2017-01-13 13:10:10,039 [INFO ]  Repository https://github.com/olipo186/Git-Auto-Deploy.git successfully cloned
2017-01-13 13:10:10,053 [CRITI]  Unable to activate SSL: File does not exist: /root/cert.pem
2017-01-13 13:10:10,098 [INFO ]  Listening for connections on http://0.0.0.0:8001`

**TL;DR;** Wrong relative load of module `.models` in files `gitautodeploy/cli/__init__.py` and `gitautodeploy/cli/config.py`. Must be `..models`.

**Comment by olipo186** [comment]
Thanks for the info, I will do as you suggest 👍
